core/domain/exp_services.py

```diff
@@ -807,6 +807,9 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
 
     exploration.version += 1
 
+    # Update summary of changed exploration.
```

Comment:
Sorry, I don't understand the first point. It doesn't look like this call used to live in save_new_exploration() I don't see it showing up in the diff.

Yes, I am concerned about linking stuff up too much. We usually do the various post-commit hooks at the level of the public method; this is the pattern the others have been following, too.

Mmm. I think in order to do this properly the refactor is necessary, I'm afraid. Do all the other tests need to track contributors correctly? If not, you could just do the modification for the ones that do need to do so.